### ISSUE TITLE
Add intermediate parameter estimates and an accuracy score as a progress hook during calibration

### ISSUE THREAD

**Issue by Tom-Szendrey** [ciemss/pyciemss#362]
Add parameter estimates and an accuracy score to the intermediate results provided by calibration

**Comment by djinnome** [comment]
@JosephCottam  can you take this task on?

**Comment by SamWitty** [comment]
@djinnome and @anirban-chaudhuri , it appears that this issue was closed with #365 . This issue is about providing parameter estimates and an accuracy score during calibration, not a progress bar for optimization.

**Comment by anirban-chaudhuri** [comment]
That is true. It seems I didn't get a reply and forgot to unlink it. It got automatically closed when my PR was merged. I will unlink my PR from this.
